### PR TITLE
Cookiecutter: Sluggify updated templates with existing variables

### PR DESCRIPTION
**Pull request by natalialuzuriaga**
## Problem

Throughout the past couple of weeks, we have been making extensive updates to markdown templates in all tiers. We would like to sluggify the updated templates with existing variables from cookiecutter.json.

## Solution

README.md
- Added `project_name` and `project_description` variables at top

CONTRIBUTING.md
- Updated branches with `project_org` and `project_repo_name` variables in Workflow and Branching section
- Updated links with `project_org` and `project_repo_name` variables in Documentation section
- Added `project_name` variable

COMMUNITY_GUIDELINES
- Added `project_name` variable

### CODE REVIEW

File: tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ All community members are expected to adhere to our [Code of Conduct](CODE_OF_CO
 
 Information on contributing to this repository is available in our [Contributing file](CONTRIBUTING.md).
 
-When participating in {Project Name} open source community conversations and spaces, we ask individuals to follow the following guidelines:
+When participating in {{ cookiecutter.project_name }} open source community conversations and spaces, we ask individuals to follow the following guidelines:
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md:23]: COMMUNITY_GUIDELINES.md is meant for external communities, and prolly could use a few tweaks for an innersource-style project. Comments below.

File: tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md [hunk 2]
```diff
@@ -20,13 +20,13 @@ All community members are expected to adhere to our [Code of Conduct](CODE_OF_CO
...
 
 - When joining a conversation for the first time, please introduce yourself by providing a brief intro that includes:
   - your related organization (if applicable)
   - your pronouns
   - disclosure of any current or potential financial interest in this work
```
Comment (decause-gov) [inline tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md:28]: I believe we can remove this line about financial interests, this concern is covered by other existing CMS/Federal policies.

**Review by decause-gov — APPROVED** [review]
Added a few minor tweaks to Tier2 that slipped by previously, but once those are addressed, everything else looks great!

LGTM +1 🚢 🍪 

After we've shipped the changes, hopefully we can find time to do an updated screencast recording of cookiecutter running that we can share for a demo at OSSNA and beyond?

File: tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md (hunk 2 above)
Comment (natalialuzuriaga) [reply tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md:28]: Should I remove this line in COMMUNITY_GUIDELINES.md across all tiers?

Comment (natalialuzuriaga) [reply tier2/{{cookiecutter.project_slug}}/COMMUNITY_GUIDELINES.md:28]: Nvm, this is a yes!

**Pull request merged**